src/rt/dmain2.d

```diff
@@ -477,6 +474,13 @@ extern (C) int _d_run_main(int argc, char **argv, MainFunc mainFunc)
             result = (result == EXIT_SUCCESS) ? EXIT_FAILURE : result;
     }
 
+    import rt.config : rt_configOption;
+
+    if (rt_configOption("noTrapExceptions") !is null)
```

Comment:
Gladly, but that means the default will be changed, right? Is that acceptable @MartinNowak